src/probnum/random_variables/_dirac.py

```diff
@@ -5,17 +5,11 @@
 import numpy as np
 
 from probnum import utils as _utils
-from probnum.type import (
-    ShapeType,
-    # Argument Types
-    ArrayLikeGetitemArgType,
-    RandomStateArgType,
-    ShapeArgType,
-)
+from probnum.type import ArrayLikeGetitemArgType  # Argument Types
+from probnum.type import RandomStateArgType, ShapeArgType, ShapeType
```

Comment:
I have the impression that here the automatic formatting didn't work perfectly, since the way I interpret this part of the code is that the `# Argument Types` comment refers to line 9, not 8. Could you change it in a way to keep the original structure while being isort compliant?

Reply:
I am not sure it's possible to configure `isort` to work that way. There is one option that is somewhat related - see [here ](https://github.com/PyCQA/isort#auto-comment-import-sections)- but I think it's a bit overkill. 

Is it possible to just remove the comment?

Reply:
I see, I also didn't find an easy way to keep the structure with the comment as before. I think removing the comment is a good idea and leads to the cleanest solution.